### ISSUE TITLE
Middleware completition doesn't recognize sanctum

### ISSUE THREAD

**Issue by kaju74** [laravel-idea/plugin#343]
Hi,

I'm using the "Sanctum" package for (api) token authentication, but the middleware entry to use isn't recognized by the plugin:

```
Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
    return $request->user();
});
```

After pressing "CTRL+SPACE", the plugin only displays:

auth, auth.basic, auth:api, auth:web

Regards,---

**Comment by adelf** [comment]
Thank you. It will be added by the next update.

**Comment by adelf** [comment]
Implemented in the 4.4 version.
Full list of changes - https://laravel-idea.com/blog/2021/4-4-release
